app/views/admin/events/index.html.haml

```diff
@@ -99,10 +101,12 @@
                 %i Hidden
             %td
               - if @program.show_voting?
-                - if speaker = event.speakers.first
-                  = link_to speaker.name, admin_user_path(speaker)
-                - else
-                  Unknown speaker
+                - event.speakers_ordered.each do |speaker|
+                  .speaker
+                    - if speaker
+                      = link_to speaker.name, admin_user_path(speaker)
+                    - else
```

Comment:
this was a leftover from the previous version of the code, I've removed these extra conditions completely